### DISCUSSION TITLE
User consent taken for granted

### DISCUSSION THREAD

**Discussion by manu0401** [rustdesk/rustdesk#4957] [General]
### Bug Description

When taking control of an Ubuntu 22.04.1 LTS, the Rustdesk window appear saying that someone is wanting to take control. There are two buttons, to accept or reject the connection, but after a few seconds without doing anything, access is granted, and the two buttons are replace by a single button to disconnect the remote user.

Why does it fall to wait for user consent? Is it a known bug? A configurable feature?

### How to Reproduce

Just take control of a remote host

### Expected Behavior

I expect RustDesk to wait for remote user consent before granting control

### Operating system(s) on local side and remote side

MacOSX 13.4 => Ubuntu 22.04.1 LTS

### RustDesk Version(s) on local side and remote side

1.1.9 -> 1.1.9

### Screenshots

none

**Comment by gezusk** [comment]
I prefer that myself. If I'm needing to work on a system, the best time is when the user is away, and not using the system.

**Reply by manu0401** [reply]
It really depends on the user and the context. You cannot take for granted that all users at any time will be at ease with their machine being taken over.

**Comment by manu0401** [comment]
Reading the source, I suspect the feature I asked about is privacy mode, and that it is only available on Windows. Am I right?

#[inline]
pub fn is_privacy_mode_supported() -> bool {
    #[cfg(windows)]
    return *IS_CAPTURER_MAGNIFIER_SUPPORTED
        && get_version_number(&crate::VERSION) > get_version_number("1.1.9");
    #[cfg(not(windows))]
    return false;
}

**Comment by manu0401** [comment]
2.1.2 addresses the problem.
